tests/Metadata/FactoryTest.php

```diff
@@ -31,7 +31,7 @@ class FactoryTest extends TestCase
      */
     public function testLoad(string $class, string $role): void
     {
-        $clientStorage = static::loadFixtureIntoMemory('Delegated');
+        $clientStorage = static::loadFixtureIntoMemory('Delegated/consistent');
```

Comment:
The metadata tests are lower-level than the updater tests, so it made sense to have them continue to use the same exact fixtures (i.e., the consistent variants) they were using before.